crossterm_cursor/src/cursor.rs

```diff
@@ -27,7 +27,9 @@ mod winapi_cursor;
 trait ITerminalCursor: Sync + Send {
     /// Goto some location (x,y) in the context.
     fn goto(&self, x: u16, y: u16) -> Result<()>;
-    /// Get the location (x,y) of the current cursor in the context
+    /// Get the location (x,y) of the current cursor in the context.
```

Comment:
"in the context"? Sounds a bit weird, can we change that to
```suggestion
    /// Get the cursor location (x,y) in the current terminal window.
```

Reply:
Updated with couple of other changes.